src/App.tsx

```diff
@@ -1,61 +1,119 @@
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
```

Comment:
Ensure that the imported files `usersFromServer` and `todosFromServer` exist and are correctly implemented in the project. This is crucial for the functionality of the app.